### PR TITLE
add in bastion host

### PR DESCRIPTION
**Pull request by karthikshivaji**

### CODE REVIEW

File: modules/bastion/main.tf [hunk 1]
```diff
@@ -0,0 +1,47 @@
+provider "aws" {
+  region = var.region
```
Comment (wneyin) [inline modules/bastion/main.tf:2]: add region

**Review by wneyin — COMMENTED** [review]
Add region

File: modules/bastion/main.tf (hunk 1 above)
Comment (karthikshivaji) [reply modules/bastion/main.tf:2]: not applicable

**Comment by wneyin** [comment]
All looks good. Please do a merge

**Pull request merged**